client/src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -27,6 +27,7 @@
       padding-right: .5rem;
       padding-left: .5rem;
       color: #ced3d9;
+      // Not a font-icon reference, it's a literal slash
       content: "/";
```

Comment:
I've added these comments in a few places because otherwise it's not immediately obvious whether it's updating an icon reference (and font-family was set elsewhere) or if it's a literal character with a regular font.